packages/@aws-cdk/aws-logs/test/destination.test.ts

```diff
@@ -65,4 +65,24 @@ describe('destination', () => {
       TargetArn: 'arn:bogus',
     });
   });
+
+  test('fromDestinationArn', () => {
```

Comment:
We also need an integ test that verifies the cross account behavior.

Reply:
Would it be ok to pull in the dependencies required to execute an integration test even if those dependencies cause a circular dependency chain?

To properly create a destination, we'll have to create a Lambda function and its associated resources.

Reply:
Also, do we have an existing integration test that uses multiple accounts? As I understand it, the integration test runner only executes from within the context of one account. How could I setup a multi-account integration test?

Reply:
we do indeed support cross account integ tests. Check out this example in `core`: https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/core/test/cross-environment-token.test.ts#L9

Reply:
You should not have to create circular dependencies in the integ tests. This likely indicates that the test belongs in the integration module, `aws-logs-destinations`, instead.